tests/em/static/test_DC_jvecjtvecadj.py

```diff
@@ -130,7 +130,8 @@ def test_misfit(self):
                 self.survey.dpred(m), lambda mx: self.p.Jvec(self.m0, mx)
             ],
             self.m0,
-            plotIt=False
+            plotIt=False,
+            num=3
```

Comment:
@sgkang: I fixed the number of iterations this runs. If we let it go too far it fails b/c we hit machine precision.